migration/v0.30/apply/90-psp-violations.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+declare -a results
```

Comment:
Since you are using the `results_mult_uniq` globally, you should change this to
```suggestion
declare -a results_mult_uniq
```